app/models/manageiq/providers/vmware/cloud_manager/vm/operations/power.rb

```diff
@@ -0,0 +1,25 @@
+module ManageIQ::Providers::Vmware::CloudManager::Vm::Operations::Power
+  def validate_pause
+    validate_unsupported("Pause operation")
+  end
+
+  def raw_start
+    with_provider_object(&:power_on)
+    self.update_attributes!(:raw_power_state => "on")
```

Comment:
It should get picked up by the UI immediately, maybe try refreshing the page once it finishes?  You can also confirm with the rails console.